packages/bridge-ui/scripts/copy-abi.js

```diff
@@ -0,0 +1,39 @@
+import fs from 'fs';
+import path from 'path';
+import { fileURLToPath } from 'url';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
```

Comment:
should we deleted all old files before copying?

Reply:
They'll be overwritten, but yes, that's a good idea in case of change of name